packages/icons/src/ui.js

```diff
@@ -133,6 +133,7 @@ import ShareAndroid from './svg/share_android.svg';
 import ShareIos from './svg/share_ios.svg';
 import ShieldCheck from './svg/shield_check.svg';
 import ShieldSecure from './svg/shield_secure.svg';
+import Shine from './svg/shine.svg';
```

Comment:
On the `Yoga` project this icon is defined under the `Filled` icons, even if we do not have a `not filled` version now it is probably better to create it as `Filled` and let the name `Shine` for the not filled one, WDYT?

```suggestion
import ShineFilled from './svg/shine_filled.svg';
```

Reply:
Done!